packages/contracts/test/test-helpers/data/index.ts

```diff
@@ -1,5 +1,38 @@
 import { create2Tests } from './create2.test.json'
 import { rlpTests } from './rlp.test.json'
+import * as fs from 'fs'
+
+const createSynthetixJSON = () => {
+  const files = {}
+  const sanitizeLibs = (str: string): string => {
+    return str
+      .split('__$')
+      .join('000')
+      .split('$__')
+      .join('000')
+  }
+  const dir = __dirname + '/synthetix/optimized/'
+  console.log(dir)
+  fs.readdirSync(dir).forEach((fileName) => {
+    if (fileName.endsWith('.json')) {
+      const obj = JSON.parse(fs.readFileSync(dir + fileName, 'utf8'))
```

Comment:
You can `require` json files directly into objects, would make this line less complex